README.md

```diff
@@ -93,7 +98,7 @@ Additional Commands:
 
 Flags:
   -C, --ci            Disable realtime terminal updates and color suitable for CI and scripting (default false)
-  -F, --from string   Comma-separated list of location values to match against. For example the partial or full name of a continent, region (e.g eastern europe), country, US state, city or network (default "world"). (default "world")
+  -F, --from string   Measurement ID or comma-separated list of location values to match against. For example the partial or full name of a continent, region (e.g eastern europe), country, US state, city or network (default "world"). (default "world")
```

Comment:
I'd swap the order everywhere because the other option is more common, e.g. "Comma-separated list of location values to match against or measurement ID"